_data/sites.json

```
@@ -13001,6 +13001,16 @@
         ]
     },
 
+    {
+        "name": "npm",
+        "url": "https://www.npmjs.com/settings/~/profile/",
+        "difficulty": "easy",
+        "notes": "Go to [your account settings](https://www.npmjs.com/settings/~/profile) and press 'Delete your account'.",
```

```suggestion
        "notes": "Go to your account settings and press 'Delete your account'.",
```
Since it is the same link as above, we can omit it.